docs/fundamentals/write-operations.txt

```diff
@@ -0,0 +1,489 @@
+      :visible: false
+
+      {
+        "_id": "660c...",
+        "performer": "Jon Batiste",
+        "venue": "Radio City Music Hall",
+        "genres": [
+          "R&B",
+          "soul"
+        ],
+        "ticketsSold": 4000,
+        "updated_at": ...
+      }
+
+.. _laravel-modify-documents-arrays:
+
+Update Arrays in a Document
+---------------------------
+
+In this section, you can see examples of the following operations that
+update array values in a MongoDB document:
+
+- Add values to an array
+- Remove values from an array
+- Update the value of an array element
+
+These examples modify the sample document created by the following insert
+operation:
+
+.. literalinclude:: /includes/fundamentals/write-operations/WriteOperationsTest.php
+   :language: php
+   :dedent:
+   :start-after: begin array example document
+   :end-before: end array example document
+
+.. _laravel-modify-documents-add-array-values:
+
+Add Values to an Array Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This section shows how to use the ``push()`` method to add values to an array
+in a MongoDB document. You can pass one or more values to add and set the
+optional parameter ``unique`` to ``true`` to skip adding any values
+in the array. The following code example shows the structure of a ``push()``
+method call:
+
+.. code-block:: none
+   :copyable: false
+
+   YourModel::where(<match criteria>)
+      ->push(
+          <field name>,
+          [<values>], // array or single value to add
+          unique: true); // whether to skip existing values
+
+The following example shows how to add the value ``baroque`` to
+the ``genres`` array field of a matching document. Click the
+:guilabel:`VIEW OUTPUT` button to see the updated document:
+
+.. io-code-block::
+
+   .. input:: /includes/fundamentals/write-operations/WriteOperationsTest.php
+      :language: php
+      :dedent:
+      :start-after: begin model array push
+      :end-before: end model array push
+
+   .. output::
+      :language: json
+      :visible: false
+
+      {
+        "_id": "660eb...",
+        "performer": "Mitsuko Uchida",
+        "genres": [
+            "classical",
+            "dance-pop",
+            "baroque"
+        ],
+        "updated_at": ...,
+        "created_at": ...
+      }
+
+
+.. _laravel-modify-documents-remove-array-values:
+
+Remove Values From an Array Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This section shows how to use the ``pull()`` method to remove values from
+an array in a MongoDB document. You can pass one or more values to remove
+from the array. The following code example shows the structure of a
+``pull()`` method call:
+
+.. code-block:: none
+   :copyable: false
+
+   YourModel::where(<match criteria>)
+      ->push(
+          <field name>,
+          [<values>]); // array or single value to remove
+
+The following example shows how to remove array values ``"classical"`` and
+``"dance-pop"`` from the ``genres`` array field. Click the
```

Comment:
S: [nit] since single quotes are used in the code:
```suggestion
The following example shows how to remove array values ``'classical'`` and
``'dance-pop'`` from the ``genres`` array field. Click the
```